theano/tensor/elemwise.py

```diff
@@ -1179,7 +1186,7 @@ def c_support_code_apply(self, node, nodename):
         return support_code
 
     def c_code_cache_version_apply(self, node):
-        version = [11]  # the version corresponding to the c code in this Op
+        version = [12]  # the version corresponding to the c code in this Op
```

Comment:
Yes and no.  I change the conditions under which we use C code and if wrong code was already compiled before, this will force it to not be used (which is a problem I had).